### PR TITLE
GUI 1637 Added support for AWS template import.

### PR DESCRIPTION
**Pull request by dkavanagh**
https://eucalyptus.atlassian.net/browse/GUI-1637
https://eucalyptus.atlassian.net/browse/GUI-1771

### CODE REVIEW

File: tests/stacks/test_aws_convert.py [hunk 1]
```diff
@@ -0,0 +1,460 @@
...
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import unittest
+import simplejson as json
+
+class TestAWSConvert(unittest.TestCase):
+    def test_aws_convert(self):
+        from eucaconsole.views.stacks import StackWizardView
+        # this is an actual aws template that contains RDS resources
+        template = """
```
Comment (kamalgill) [inline tests/stacks/test_aws_convert.py]: @dkavanagh: We should probably move this template to an external file and have the unit test read from that file

**Comment by kamalgill** [comment]
Hal, retest this please

File: eucaconsole/views/stacks.py [hunk 2]
```diff
@@ -29,20 +29,23 @@
 
 """
 import simplejson as json
+from simplejson import JSONDecodeError
```
Comment (gholms) [inline eucaconsole/views/stacks.py]: This exception is not available in the version of simplejson that ships with RHEL 6.

**Comment by aliceh** [comment]
Good to merge. Random SeleniumRC glitch.

**Comment by kamalgill** [comment]
This PR is also for https://eucalyptus.atlassian.net/browse/GUI-1649

**Comment by kamalgill** [comment]
Hal, retest this please

**Comment by dkavanagh** [comment]
hal, retest this please

File: eucaconsole/static/js/pages/stack_wizard.js [hunk 3]
```diff
@@ -137,6 +154,28 @@ angular.module('StackWizard', ['TagEditor', 'EucaConsoleUtils', 'localytics.dire
                 $scope.checkRequiredInput();
             });
             $scope.$watch('inputtype', function(){
```
Comment (kamalgill) [inline eucaconsole/static/js/pages/stack_wizard.js]: @dkavanagh: No need to change it here, but the function can be spec'd  as `function(newValue, oldValue)` , where newValue is $scope.inputtype (so the switch statement would be `switch (newValue) {... }`.  
See the example at https://docs.angularjs.org/api/ng/type/$rootScope.Scope#example

**Comment by kamalgill** [comment]
The Jenkins failures have nothing to do with the code changes here, so this should be safe to merge.

**Pull request merged**